package.json

```diff
@@ -2,7 +2,7 @@
     "name": "azurerm-vscode-tools",
     "displayName": "Azure Resource Manager (ARM) Tools",
     "description": "Language server, editing tools and snippets for Azure Resource Manager (ARM) template files.",
-    "version": "0.8.5-alpha",
+    "version": "0.9.0-alpha",
```

Comment:
>9 [](start = 18, length = 1)

Nit: Can we go past 1.0 without going GA?  Are we incrementing too much?

Reply:
It goes to 0.10.0 next.